hack/e2e.sh

```diff
@@ -106,6 +84,11 @@ function test_elasticsearchcluster_create() {
             ElasticSearchClusters; then
         fail_test "Failed to get elasticsearchclusters"
     fi
+    if ! retry kubectl get \
+         --namespace "${USER_NAMESPACE}" \
+         service es-demo; then
+        fail_test "Navigator controller failed to create elasticsearchcluster service"
+    fi
```

Comment:
Added a check to see that a service is created....a clear indication that the navigator controller is reacting to the new escluster resource.